.gitignore

```diff
@@ -39,17 +39,7 @@ captures/
 
 # IntelliJ
 *.iml
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/gradle.xml
-.idea/assetWizardSettings.xml
-.idea/dictionaries
-.idea/libraries
-.idea/caches
-# Android Studio 3 in .gitignore file.
-.idea/caches/build_file_checksums.ser
-.idea/modules.xml
-.idea/codeStyles/
+.idea
```

Comment:
Should this be something like
`.idea/*`
?

Reply:
it seemed to work and this is what's in MSAL so I used that as inspiration